src/components/CreateCourseRunPage/index.jsx

```diff
@@ -138,7 +138,8 @@ class CreateCourseRunPage extends React.Component {
     const parsedTypeOptions = courseOptionsData && courseOptionsData.type &&
       parseCourseTypeOptions(courseOptionsData.type.type_options);
     const courseRunTypeOptions = parsedTypeOptions && parsedTypeOptions.courseRunTypeOptions;
-    const defaultRunType = courseRunTypeOptions && courseRunTypeOptions[type][1].value;
+    const defaultRunType = courseRunTypeOptions && courseRunTypeOptions[type] &&
+      courseRunTypeOptions[type][1].value;
```

Comment:
Do you also want to add a `&& courseRunTypeOptions[type][1]`?

Reply:
Can't hurt. Going to update test too